src/win32.mak

```diff
@@ -377,8 +380,11 @@ pvs:
 #	$(PVS) --cfg PVS-Studio.cfg --cl-params /I$C;$(TK) /Tp $(BACKSRC) --source-file $(BACKSRC)
 #	$(PVS) --cfg PVS-Studio.cfg --cl-params /I$(TK) /Tp $(TKSRCC) --source-file $(TKSRCC)
 
-checkwhitespace:
-	$(HOST_DC) -run checkwhitespace $(SRCS) $(GLUESRC) $(ROOTSRC)
+checkwhitespace: $(TOOLS_DIR)\checkwhitespace.d
+	$(HOST_DC) -run $(TOOLS_DIR)\checkwhitespace $(SRCS) $(GLUESRC) $(ROOTSRC)
+
+$(TOOLS_DIR)\checkwhitespace.d:
+	git clone --depth=1 $(GIT_HOME)/tools $(TOOLS_DIR)
```

Comment:
This fails on Windows if the tools directory already exists: DigitalMars make executes the rule unconditionally.